nix/runtime.nix

```diff
@@ -7,8 +7,8 @@ rec {
     # See `doc/development.md` and `doc/runtime.md#changing-network-configurations`
     # for info on how to switch networks.
     network = {
-      name = "preview";
-      magic = 2; # use `null` for mainnet
+      name = "preprod";
+      magic = 1; # use `null` for mainnet
```

Comment:
Let's keep a change request here to prevent this from being merged

Reply:
This change was to check locally? @klntsky said that we cannot change this network.

Reply:
Yes, we should remove this change after testing is done by reviewers.